Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -627,6 +1249,19 @@ def main():  # pragma: no cover
             else:
                 demisto.info(f"Got less than {limit} events this interval - will not trigger next run automatically.")
             demisto.setLastRun(next_run)
+        elif command == "long-running-execution":
```

Comment:
add a validation on this command and on the fetch-events command that they are not both enabled, document it